.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
don't think we need this to be committed not everybody is using VS Code...

Reply:
Theres a whole vscode section in .gitignore 

# IDE - VSCode
.vscode/*
!.vscode/settings.json
!.vscode/tasks.json
!.vscode/launch.json
!.vscode/extensions.json
.history/*

Is this not being read